interfaces/CMakeLists.txt

```diff
@@ -69,7 +62,7 @@ if (WRAP_SOQT)
         ${CMAKE_SOURCE_DIR}/interfaces
         )
 
-    target_link_libraries(soqt SoQt)
+    target_link_libraries(soqt SoQt::SoQt)
     set_target_properties(soqt PROPERTIES LIBRARY_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/pivy/gui")
     install(TARGETS soqt DESTINATION ${python_libs}/pivy/gui)
-endif(WRAP_SOQT)
+endif(SoQt_FOUND)
```

Comment:
`endif(SoQt_FOUND)` is old style (CMake < 2.8) and can be changed to `endif()`

Reply:
Sometimes it makes sense to have the condition also in the endif. But here it doesn't make much sense, so I will remove that, thanks